src/olympia/activity/migrations/0004_auto_20191115_1154.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.6 on 2019-11-15 11:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('activity', '0003_auto_20191115_1151'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blocklog',
+            name='guid',
+            field=models.CharField(max_length=255),
```

Comment:
Can't do that yet. Splitting into 2 migrations that run in the same deploy window doesn't solve the problem of having potentially `BlockLog` entries being created by old code after migrations have ran but before the push is done.

Reply:
It's fine, it's fine 😉 It's not in use yet by anyone so unless QA are unlucky enough to be using django admin during the deploy window it won't affect anyone.

Reply:
Fair enough. In that case though, you don't need 2 migrations - you can set a `default` in the migration when adding the field.

Reply:
It was a separate migration so the RunPython can populate the null values before the column is changed to be actually `null=False` (or it fails db integrity checking when the column is added). 
Or are you suggesting setting it to some other non-null value?

Reply:
Just an empty string ?

Reply:
https://github.com/mozilla/addons-server/pull/12885/commits/c22bf522fc758f78aa45207927fe472fb1a454eb